### ISSUE TITLE
Traceback and duplicate tabs when repeatly delete/create/delete/create Image channel

### ISSUE THREAD

**Issue by pllim** [ejeschke/ginga#258]
When I delete channel "Image", recreate it, delete it again, I get the following traceback:

```python
2015-12-18 12:43:36,473 | E | ImageView.py:996 (redraw_now) | Error redrawing image: wrapped C/C++ object of type RenderWidgetZoom has been deleted
2015-12-18 12:43:36,474 | E | ImageView.py:1001 (redraw_now) | Traceback:
  File ".../ginga-2.5.20151215192336-py2.7.egg/ginga/ImageView.py", line 986, in redraw_now
    self.update_image()
  File ".../ginga-2.5.20151215192336-py2.7.egg/ginga/qtw/ImageViewQt.py", line 305, in update_image
    self.imgwin.update()
```

Perhaps we need a safeguard after all, similar to the unmerged #243 but check for zero number of channels instead of one?

**Comment by pllim** [comment]
I also see duplicate "Image" tabs, as attached:

![untitled](https://cloud.githubusercontent.com/assets/2090236/11903240/bef1d2ce-a585-11e5-9c3f-c8ed50e6e91d.png)

**Comment by pllim** [comment]
Update: I only see the traceback with PR #223, not master. But I still see the duplicate tabs. I'll look into the PR and see if I can put in a safeguard there.

**Comment by pllim** [comment]
Update: Still see the traceback with spacetelescope/stginga#61 when deleting the channel for the first time. In that case, it must be trying to do something unexpected with my plugins? I cannot make sense of it. I traced it to `update_image()` in `ginga/qtw/ImageViewQt.py` with the following variable values:

```python
>>> imgwin
<ginga.qtw.ImageViewQt.RenderWidgetZoom object at ...>
>>> pixmap
PyQt4.QtGui.QPixmap object at ...>
```

Anyways, it is not critical. But please let me know if you have seen it before and know what it is about. Thanks!

**Comment by pllim** [comment]
Anyways, no matter what I do, duplicate tabs are still there. I don't know how to get rid of them as I do not see anything obvious.

**Comment by ejeschke** [comment]
@pllim, I believe this is fixed in commit b561f6059bb4d73988a0189935afc7dbb90872fa, at least to my testing.  Can you confirm? Tnx

**Comment by pllim** [comment]
Thanks, @ejeschke ! Confirmed fixed.
